packages/css/src/input.css

```
@@ -34,6 +35,8 @@
   --_dsc-input-diagonal-2: currentcolor calc(50% - var(--dsc-input-stroke-width));
   --_dsc-input-diagonal-3: currentcolor calc(50% + var(--dsc-input-stroke-width));
   --_dsc-input-diagonal-4: transparent calc(52% + var(--dsc-input-stroke-width));
+  --dsc-input-line-height: var(--ds-line-height-md);
+  --_dsc-input-padding: var(--dsc-input-padding-block) var(--dsc-input-padding-inline);
```

```suggestion
  --dsc-input-padding: var(--dsc-input-padding-block) var(--dsc-input-padding-inline);
```
Renaming from `--dsc-input-padding` would be considered a breaking change

---

Yep, I didn't like it either :D Fixed.